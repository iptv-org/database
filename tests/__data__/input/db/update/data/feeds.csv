channel,id,name,is_main,broadcast_area,timezones,languages,video_format
002RadioTV.do,SD,SD,TRUE,c/DO,America/Santo_Domingo,spa,480i
01TV.fr,SD,SD,TRUE,c/FR,Europe/Paris,fra,576i
0TV.dk,SD,SD,TRUE,c/DK,Europe/Copenhagen,dan,576i
1000xHoraTV.uy,SD,SD,TRUE,c/UY,America/Montevideo,spa,576i
BeijingSatelliteTV.cn,SD,SD,TRUE,c/DO,America/Santo_Domingo,spa,480i
M5.hu,SD,SD,FALSE,c/DO,America/Santo_Domingo,spa,480i
M5.hu,West,West,TRUE,c/DO,America/Santo_Domingo,spa,480i
Channel82.bm,SD,SD,FALSE,c/BM,America/Santo_Domingo,eng,480i
TrueCrime.uk,SD,SD,TRUE,c/UK;c/IE,Europe/London,eng,576i