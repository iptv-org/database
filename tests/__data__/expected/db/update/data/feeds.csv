channel,id,name,alt_names,is_main,broadcast_area,timezones,languages,format
0TV.dk,SD,SD,,TRUE,c/DK,Europe/Copenhagen,dan,576i
1000xHoraTV.uy,HD,HD,Oeste;Este,TRUE,c/CN,Africa/Johannesburg;Africa/Kigali,zho,576i
1000xHoraTV.uy,SD,SD,,FALSE,c/UY,America/Montevideo,spa,576i
beINMoviesTurk.tr,SD,SD,,TRUE,c/DO,America/Santo_Domingo,spa,480i
Channel82.bm,SD,SD,,FALSE,c/BM,Atlantic/Bermuda,eng,480i
M5.hu,HD,HD,Méditerranée;Sud,TRUE,c/BR,Africa/Dakar;Africa/El_Aaiun,por;spa,1080i
M5.hu,West,West,,FALSE,c/DO,America/Santo_Domingo,spa,480i
TrueCrime.uk,Plus1,+1,,FALSE,c/UK,Europe/London,eng,576i
TrueCrime.uk,SD,SD,,TRUE,c/UK;c/IE,Europe/London,eng,576i
WenzhouEconomicandEducation.cn,SD,SD,,TRUE,c/CN,Africa/Johannesburg;Africa/Kigali,zho,576i
YiwuBusinessChannel.cn,SD,SD,,TRUE,c/CN,Africa/Johannesburg;Africa/Kigali,zho,576i
YiwuNewsIntegratedChannel.cn,SD,SD,,TRUE,c/CN,Africa/Johannesburg;Africa/Kigali,zho,576i
ZutTV.ve,HD,HD,,TRUE,c/UK;c/IE,Europe/London,eng,576i